python/marvin/tools/map.py

```diff
@@ -386,3 +417,257 @@ def plot(self, array='value', xlim=None, ylim=None, zlim=None,
             return (ax, fig)
         else:
             return ax
+
+    def _make_image(self, snr_min, log_cb, use_mask):
+        """Make masked array of image.
+
+        Args:
```

Comment:
We are using Google Style docstrings.  We have been using Parameters: in all the tools and functions.  Maybe that is synonymous with Args:.  But the rest of the layout is Google Style.